RocketChat/Rocket.Chat.ReactNative
# [NEW] On-Hold chats for Omnichannel

## Proposed changes

Implement chats on hold on mobile in omnichannel chats.

## Issue(s)	


## How to test or reproduce


https://docs.rocket.chat/guides/omnichannel/omnichannel-agents-guides/omnichannel-chat/omnichannel-chat-quick-actions#place-chat-on-hold

- You can place the chat on-hold manually only when the last message was sent by you to the contact.
- Do an exhausting test in RoomsListView, trying grouping the chats by types, changing avatar, and others... The re-render should be the same as usual.

## Screenshots

https://user-images.githubusercontent.com/47038980/163239366-104f4397-6e5c-454b-af14-0c94b9aed4cd.mov


## Types of changes




## Checklist



## Further comments
